### PR TITLE
CodeQL and CodeClimate (standalone) workflow configurations

### PR DESCRIPTION
**Pull request by Dlitosh**
CodeQL workflow results are available in the Security tab of the Guthub project
CodeClimate results are uploaded as HTML (and JSON) report in the root of the action (can be downloaded and observed)

### CODE REVIEW

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '16 12 * * 5'
```
Comment (commjoen) [inline .github/workflows/codeql-analysis.yml:21]: hi! thank you so much for your PR! Can you do this on pull and tag only please :) ?

File: .github/workflows/codeclimate_standalone.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# This is a basic workflow that is manually triggered
+
+name: CodeClimate Standalone workflow
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on: [push, pull_request, workflow_call]
```
Comment (commjoen) [inline .github/workflows/codeclimate_standalone.yml:7]: can you add tag over here as well please ;) ?

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (commjoen) [reply .github/workflows/codeql-analysis.yml:21]: and on workflow_call ^^

**Review by commjoen — CHANGES_REQUESTED** [review]
Hi @Dlitosh ! Thank you so much for contributing to wrongsecrets! CAn you have a look at the comments please?

File: .github/workflows/codeclimate_standalone.yml (hunk 2 above)
Comment (Dlitosh) [reply .github/workflows/codeclimate_standalone.yml]: I think I did (edited the file), though i'm not sure

**Comment by commjoen** [comment]
Waiting for the analyzers to be complete before I review again ;) . Thank you @Dlitosh !

**Comment by commjoen** [comment]
```
1 issue was detected with this workflow: Please specify an on.push hook so that Code Scanning can compare pull requests against the state of the base branch.Show less
--

**Review by commjoen — APPROVED** [review]

**Pull request merged**